test/irb/test_history.rb

```diff
@@ -14,12 +14,22 @@ def teardown
       IRB.conf[:RC_NAME_GENERATOR] = nil
     end
 
-    class TestInputMethodWithHistory < TestInputMethod
-      HISTORY = Array.new
+    class TestInputMethodWithRelineHistory < TestInputMethod
+      # When IRB.conf[:USE_MULTILINE] is true, IRB::ReidlineInputMethod uses Reline::History
+      HISTORY = Reline::History.new(Reline.core.config)
```

Comment:
Changed from an Array to the history class that is used by default when `:USE_MULTILINE` is on.